src/Gui/Dialogs/DlgPreferencesImp.cpp

```diff
@@ -1546,15 +1543,15 @@ void PreferencesSearchController::navigateToCurrentSearchResult(PopupAction acti
     }
 }
 
-void PreferencesSearchController::populateSearchResultsList()
+void PreferencesSearchController::populateSearchResultsList() const

```

Comment:
Is it const, though? It clearly changes the state of the object but deeply nested state referred by pointer. The fact that it can be made const does not mean that it always should. Methods marked as `const` should be used only for things that does not change state of the object to ensure that if we pass it as const reference to some function and try to use it later the object remains unchanged.